environments/mucoll-release/spack.yaml

```diff
@@ -5,7 +5,7 @@ spack:
     clhep:
       require: '@2.4.7.1 ~cms~ipo'
     root:
-      require: '@6.32.00 +fftw+http+mlp+pythia8+r+root7+sqlite+ssl+tmva+vc+xrootd'
+      require: '+fftw+http+mlp+pythia8+r+root7+sqlite+ssl+tmva+vc+xrootd'
```

Comment:
Does this now build the latest / expected version of root? We had to explicitly require `6.30` at some point because it would fall back to `6.28` otherwise for some reason we have not figured out.

Reply:
Seems like it does: https://gitlab.cern.ch/kkrizka/MuonCollider-docker/-/jobs/41106592#L3173